src/map/mob.c

```diff
@@ -5932,7 +5972,7 @@ static int mob_reload_sub_mob(struct mob_data *md, va_list args)
 	status_calc_mob(md, SCO_FIRST);
 
 	// If the view data was not overwritten manually
-	if (md->vd != NULL) {
+	if ( !md->vd_changed ) {
```

Comment:
if (!md->vd_changed)


(Remove the spaces around the boolean statement

Reply:
Uploaded changes